tool/README.md

```diff
@@ -139,6 +106,21 @@ From the git GUI tool or from github.com directly:
    tool/tag_version.sh;
    ```
 
+### Verify and Submit the release notes
+
+See the release notes
+[README.md](https://github.com/flutter/devtools/blob/master/packages/devtools_app/lib/src/framework/release_notes/README.md)
+for details on where to add DevTools release notes to Flutter website and how to test them.
+
+1. The release notes for the clean version can be found in [NEXT_RELEASE_NOTES.md](./release_notes/NEXT_RELEASE_NOTES.md)
+2. Follow the release notes
+[README.md](https://github.com/flutter/devtools/blob/master/packages/devtools_app/lib/src/framework/release_notes/README.md)
+to add these DevTools release notes to Flutter website
+   - make sure to also follow the instructions to test them.
```

Comment:
more of a side note out of the scope of this PR, but testing the release notes viewer in DevTools is cumbersome.